src/Voxel.MiddyNet.HttpJsonBodyParserMiddleware/HttpJsonBodyParserMiddleware.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Text;
+using Amazon.Lambda.APIGatewayEvents;
+using Newtonsoft.Json;
```

Comment:
I think we´re using system.text.json in other parts of the code. Can we use it here too?

Reply:
Yes, already change it